OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -218,13 +237,7 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world)
 
 			var closeButton = panel.GetOrNull<ButtonWidget>("CLOSE_BUTTON");
 			if (closeButton != null)
-				closeButton.OnClick = () =>
-				{
-					if (isVideoLoaded)
-						player.Stop();
-					Ui.CloseWindow();
-					onExit();
-				};
+				closeButton.OnClick = () => closeAssetBrowser();
```

Comment:
You don't need the wrapper: `closeButton.OnClick = closeAssetBrowser;` does what you want.